server/src/assembly/resources/conf/iotdb-engine.properties

```diff
@@ -505,6 +505,14 @@ timestamp_precision=ms
 # Datatype: int
 # query_timeout_threshold=60000
 
+####################
```

Comment:
Please also update doc

Reply:
This PR focuses on the code, and we need subsequent PR to supplement the detailed documentation.